pkg/scheduler/framework/plugins/interpodaffinity/scoring.go

```diff
@@ -262,7 +262,7 @@ func (pl *InterPodAffinity) NormalizeScore(ctx context.Context, cycleState *fram
 	}
 
 	var minCount int64 = math.MaxInt64
-	var maxCount int64 = -math.MaxInt64
+	var maxCount int64 = math.MinInt64
```

Comment:
> this is not equivalent
> 
> https://go.dev/play/p/v1VgeLIDZks
> 
> ```
> maxInt -9223372036854775807 minint -9223372036854775808
> ```

Yes @aojea They really are different.
Given the context, I think we should use `math.MinInt64` instead of `-math.MaxInt64` here. 
As we're looking for the maximum, if all the values are -9223372036854775808, we will find the wrong maximum -9223372036854775807

Reply:
then, if this is a bugfix, you should add a testcase covering that ^^^

Reply:
Also, if it's a bugfix, even a small one, does it need a changelog entry as well?

Reply:
This may not seem like a bug. When all the scores are -9223372036854775808, even though we used the wrong maximum -9223372036854775807, but the final scores are fine, because `-9223372036854775808 - -9223372036854775808 = 0`:
https://github.com/kubernetes/kubernetes/blob/7e2c02d4c86d73051b6d4f47298d35a7ffb42b6a/pkg/scheduler/framework/plugins/interpodaffinity/scoring.go#L280

Reply:
I'm lost here :smile: , @alculquicondor or some other sig-scheduling folks should judge this better

thanks for the explanation , though

Reply:
hahaha tricky to see that there is no bug indeed.

But better be safe and use MinInt64. Thanks!

/lgtm
/approve